1080i/MyPVRRecordings.xml

```diff
@@ -173,12 +173,20 @@
 						<font>font36_title</font>
 						<label>$INFO[ListItem.Title]</label>
 					</control>
+					<control type="label">
+						<top>240</top>
+						<width>560</width>
+						<height>262</height>
+						<label>$INFO[ListItem.EpisodeName]</label>
+						<font>font14</font>
+					</control>
 					<control type="textbox">
-						<top>254</top>
+						<top>282</top>
 						<width>750</width>
 						<height>526</height>
 						<align>justify</align>
 						<label>$INFO[ListItem.Plot]</label>
+						<textcolor>grey</textcolor>
```

Comment:
Sorry guys I was out all weekend. I love that suggestion. I'll make the change and resquash today.